quad_pyblish_module/plugins/tvpaint/create/create_playblast.py

```diff
@@ -31,7 +31,7 @@ def apply_settings(self, project_settings, system_settings):
         self.extract_psd = plugin_settings["extract_psd"]
         self.apply_background = False
         self.exports_types = ['camera', 'scene']
-        self.export_type = self.exports_types[0]
+        self.export_type = self.exports_types[1]
```

Comment:
À quoi est dû ce changement?

Reply:
S'il n'y a pas de cam dans la scene tvpp, le publish peut planter.
Donc je met par defaut scene plutot

Reply:
j'aurais je pense laissé `self.exports_types[0]` mais interverti les valeurs:
```python
self.exports_types = ['scene', 'camera']  # 'scene' need to be first to avoid crash if there isn't any camera 
```

Reply:
ça me va aussi, en soit cela va bientot disparaitre quand j'aurai fini le refacto du publish tvpaint.
C'etait temporaire le temps de finir proprement cet autre sujet